bugsnag-plugin-android-exitinfo/src/main/java/com/bugsnag/android/InternalHooks.java

```diff
@@ -1,13 +1,37 @@
 package com.bugsnag.android;
 
+import static com.bugsnag.android.Bugsnag.client;
+
 import kotlin.Unit;
 import kotlin.jvm.functions.Function0;
 
 class InternalHooks {
 
-    private InternalHooks() {}
+    public InternalHooks() {
```

Comment:
all methods in InternalHooks are static, there's no reason for it to have a `public` constructor

```suggestion
    private InternalHooks() {
```